projects/packages/videopress/src/class-initializer.php

```diff
@@ -132,6 +132,7 @@ private static function active_initialization() {
 		add_filter( 'embed_oembed_html', array( __CLASS__, 'enqueue_videopress_iframe_api_script' ), 10, 4 );
 
 		if ( self::should_initialize_admin_ui() ) {
+			add_action( 'admin_menu', array( '\Automattic\Jetpack\VideoPress\Admin_UI', 'enable_menu' ) );
```

Comment:
Any particular reason not to put this inside `Admin_UI::init()` along with the other `add_action()` calls?

Reply:
Yes it could, it is a lot cleaner, thanks for pointing that out. Initially I was seeing the menu item wasn't there, but further testing seems fine, so possibly my environment hadn't built properly.